src/packaging/packages.targets

```diff
@@ -60,13 +60,40 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/common.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.base.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.object.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.sync.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gc.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcdesc.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/sample/gcenv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcimpl.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcpriv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcrecord.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcscan.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletable.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletable.inl" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletablepriv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/objecthandle.h" />
+            <ILCompilerSdkCppCodgenCommonFiles Include="Native/Common/CPPCodegen/common.h" />
```

Comment:
I do not think there should be distinction between `ILCompilerSdkCppCodgenCommonFiles` and `ILCompilerSdkCppCodegenFiles`. There should be just `ILCompilerSdkCppCodegenFiles`.